mantidproject/mantid
# IntegrateEPP crash on workspace group

**Description of work.**
The IntegrateEPP was crashing when being run from the algorithm dialog when using a workspace group as the input workspace . Running the algorithm via the python editor was fine.

**To test:**

1. Run this script:
```
from mantid.simpleapi import *
import matplotlib.pyplot as plt
import numpy as np

gaussian = 'name=Gaussian, PeakCentre=7000, Height=230, Sigma=680'
ws = CreateSampleWorkspace('Histogram', 'User Defined', gaussian)
ws1 = CreateSampleWorkspace('Histogram', 'User Defined', gaussian)
wsGroup = GroupWorkspaces([ws,ws1])
epps = FindEPP(ws)

integrated = IntegrateEPP(wsGroup, epps, 3)
```
2. Attempt to run the IntegrateEPP algorithm via the GUI, giving it the same input as the script, but a different output workspace name.
3. Verify that the algorithm runs correctly by checking that the output is consistent with the "integrated" ws group that was output from the python script.


Fixes #28705 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
